irc/src/connection/tls.rs

```diff
@@ -30,7 +30,7 @@ pub async fn connect<'a>(
         let mut roots = rustls::RootCertStore::empty();
 
         for cert in rustls_native_certs::load_native_certs()? {
-            roots.add(cert).unwrap();
+            let _ = roots.add(cert);
```

Comment:
I don't think it's necessary to log the failed DER certs, so for the first proposal, I’ll just ignore the result instead.